crates/rpc/src/handlers/config.rs

```diff
@@ -0,0 +1,24 @@
+use alloy_primitives::Address;
+use serde::{Deserialize, Serialize};
+use warp::{
+    http::status::StatusCode,
+    reject::Rejection,
+    reply::{Reply, with_status},
+};
+
+use super::Data;
+
+#[derive(Serialize, Deserialize, Default)]
+struct DepositContract {
+    chain_id: u64,
```

Comment:
```suggestion
    #[serde(with = "serde_utils::quoted_u64")]
    chain_id: u64,
```

this is currently be serialized wrong

Reply:
Thanks for this catch.